src/common/config.cc

```diff
@@ -204,8 +204,18 @@ int md_config_t::parse_config_files(const char *conf_files,
 				    int flags)
 {
   Mutex::Locker l(lock);
+
   if (internal_safe_to_start_threads)
     return -ENOSYS;
+
+  if (!cluster.size() && !conf_files) {
```

Comment:
Appreciate your comments, I change it to:
===%<====
 if (cluster.size() == 0) 
==%<=====
 it would be more clear;)
And I changed the commit comments to：default cluster name to config file prefix